docs/README.md

```diff
@@ -998,9 +998,80 @@ _Reference_
 - https://www.freecodecamp.org/news/javascript-new-features-es2020/
 
 
+## 19. Iterators 
 
-<br/> <br/>  <br/>
+Iterators provide mechanism to iterate over collections.
 
```

Comment:
"An iterable is an object that can return an iterator using the iter() function; while an iterator is an object that keeps state and produces the next value when you call the next() function on it"